src/Stripe.Tests.XUnit/Stripe.Tests.XUnit.csproj

```diff
@@ -27,4 +27,8 @@
     <ProjectReference Include="..\Stripe.net\Stripe.net.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I can, but VS is just going to add it back in.   Per Microsoft, it's intended behavior.  https://connect.microsoft.com/VisualStudio/feedback/details/800245/vs2013rc-adds-to-vs2012-c-project-section-itemgroup  

>This behavior is intentional. 
>
>To support third-party test frameworks, like NUnit and XUnit, Visual Studio 2012 
>loaded Test Explorer on solution open, regardless of whether it contained test 
>projects. This added seconds of delay to startup and solution open scenarios for all 
>users, majority of whom don't use tests.
>
>In Visual Studio 2013, we changed it so that Test Explorer package is loaded only 
>when the solution contains one or more test projects. Test projects are identified in 
>two different ways. Projects created from one of the built-in unit test project 
>templates are identified using project type GUIDs. Other types of projects, such as 
>Class Library project with XUnit or NUnit tests, are identified by Test Explorer 
>during first test discovery and “tagged” with the <Service/> item. 
>
>You can avoid Visual Studio automatically adding the <Service/> item to your test 
>projects by creating them from any of the built-in unit test project templates. In 
>other words, select the "Unit Test Project" template instead of the "Class Library" 
>template in the New Project dialog.